app/src/main/java/com/wizeline/compose/academy/components/CircleIconButton.kt

```diff
@@ -0,0 +1,38 @@
+package com.wizeline.compose.academy.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.FavoriteBorder
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun CircleIconButton(
+    icon: ImageVector
+){
+    Box(
+        modifier = Modifier.padding(10.dp)
```

Comment:
Only a friendly reminder that you should move your dimens into a XML or Object class 🐸